lib/packageManagers.ts

```diff
@@ -14,13 +26,18 @@ export function detectPackageManager(directoryPath: string): PackageManager {
     return `npm`;
 }
 
-export function getInstallCommand(packageManager: PackageManager) {
-    switch (packageManager) {
-        case 'npm':
-            return 'npm ci';
-        case 'pnpm':
-            return 'pnpm install --frozen-lockfile';
-        case 'yarn':
-            return 'yarn install --frozen-lockfile';
+export function getInstallCommand(
+    packageManager: PackageManager,
+    debug: boolean = false,
```

Comment:
I think we can just import the global debug environment variables instead of passing it in as a parameter like this. Check out how it is setup in `injectCfnRole.ts` file:
```typescript
import { env } from './env';

if (env.debug) {
            console.log(
                logSymbols.info,
                chalk.whiteBright(JSON.stringify(serverless, null, 2))
            );
        }
```

Reply:
Good pickup! Thanks, I'll update.